docs/README.md

```diff
@@ -1,3 +1,4 @@
+<!-- FIXME: what is the purpose of this README? -->
```

Comment:
Is this file required?

Reply:
That is a good idea!
It could also tell it is based on Jekyll